app/presenters/line_mailchimp_presenter.rb

```diff
@@ -14,8 +14,8 @@ def json
           id: Digest::MD5.hexdigest("#{line.id}#{line.order_id}"),
           product_id: ProductMailchimpPresenter.new(line.product).json["id"],
           product_variant_id: Digest::MD5.hexdigest("#{line.variant.sku}#{line.variant.id}"),
-          quantity: line.quantity,
-          price: line.price.to_s
+          quantity: line.quantity || 0,
+          price: line.price.to_s || 0
```

Comment:
`line.price.to_s` will return a string, but `0` will be an integer - is this the way it should be?